actors/builtin/miner/policy.go

```diff
@@ -92,15 +92,15 @@ const ElectionLookback = abi.ChainEpoch(1) // PARAM_FINISH
 // This lookback exists so that deadline windows can be non-overlapping (which make the programming simpler)
 // but without making the miner wait for chain stability before being able to start on PoSt computation.
 // The challenge is available this many epochs before the window is actually open to receiving a PoSt.
-const WPoStChallengeLookback = abi.ChainEpoch(20)
+const WPoStChallengeLookback = abi.ChainEpoch(16) // 8 minutes
 
 // Minimum period before a deadline's challenge window opens that a fault must be declared for that deadline.
 // This lookback must not be less than WPoStChallengeLookback lest a malicious miner be able to selectively declare
 // faults after learning the challenge value.
-const FaultDeclarationCutoff = WPoStChallengeLookback + 50
+const FaultDeclarationCutoff = WPoStChallengeLookback + 42 // 29 minutes
```

Comment:
This used to be 29.1666 minutes.

Also, this is awfully close to one full deadline. Should this just be a full deadline?

Reply:
This should still be 50.

Reply:
This increases the fault declaration cutoff from 29 minutes (slightly less than a full deadline ahead) to 35 minutes (more than a deadline). Is that OK?